docs/source/commands.rst

```diff
@@ -33,3 +35,39 @@ Command-line options
 .. option:: --app-label <app_label>
 
    Create initial data with the *app_label* provided.
+
+.. _command-generate-pep-pages:
+
+generate_pep_pages
+------------------
+
+This Django management command generates `pages.Page` objects from the output
```

Comment:
I know that you just copied this from `docs/source/pep_generation.rst`, but we can drop the "Django management" part.

Reply:
Style nit: \`pages.Page\` -> \`\`pages.Page\`\`